src/db.js

```
@@ -0,0 +1,16 @@
+import { Sequelize } from 'sequelize';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const sequelize = new Sequelize(
+  process.env.DB_NAME,
+  process.env.DB_USER,
+  process.env.DB_PASSWORD,
+  {
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    dialect: 'postgres',
+    logging: false,
+  },
+);
```

While the Sequelize instance is configured correctly, there's no check to ensure the connection is valid before the application starts. If the database credentials are wrong or the server is down, the app will crash on the first database query. It's a good practice to call `sequelize.authenticate()` at application startup (e.g., in `index.js`) to verify the connection and fail early with a clear error message.